src/components/BuildConfigCreateEditPage/BuildConfigCreateEditPage.tsx

```diff
@@ -910,10 +910,10 @@ export const BuildConfigCreateEditPage = ({ isEditPage = false }: IBuildConfigCr
 
               {Object.entries(buildParamData).map(([key, buildParam], index) => (
                 <FormInput<string>
+                  key={index}
```

Comment:
In React 18, because of changes in how rendering works, React now requires that every item in a list has a stable and unique key. Before, the key was set on the inner FormGroup, but the top-level element returned from the map was the FormInput component. Thus I have to move the key to the outer element (the FormInput).